config/logger/logger.js

```diff
@@ -0,0 +1,30 @@
+// Package imports
+const {createLogger, format, transports} = require('winston');
+
+const logger = createLogger({
+  level: 'info',
+  format: format.combine(
+      format.timestamp({
+        format: 'YYYY-MM-DD HH:mm:ss',
+      }),
+      format.errors({stack: true}),
+      format.splat(),
+      format.json(),
+  ),
+  defaultMeta: {service: 'user-service'},
+  transports: [
+    new transports.File({filename: 'error.log', level: 'error'}),
+    new transports.File({filename: 'quick-start-combined.log'}),
+  ],
```

Comment:
```suggestion
defaultMeta: {service: 'user-service'},
  transports: [
    new transports.File({filename: 'error.log', level: 'error'}),
    new transports.File({filename: 'quick-start-combined.log'}),
  ],
```
Explain this.

Reply:
it is the metadata that would be included in all the logger messages. I added it just to see how it works

Reply:
Done